torch/_dynamo/convert_frame.py

```diff
@@ -1177,7 +1179,9 @@ def __call__(
                         frame, cache_entry, self.hooks, frame_state
                     )
 
-        with compile_lock, _disable_current_modes():
+        with (
+            compile_lock
+        ), _disable_current_modes(), torch.fx.experimental._sym_dispatch_mode.disable_sym_dispatch():
```

Comment:
I extracted this into https://github.com/pytorch/pytorch/pull/132433

Reply:
Delete?